aws/rust-runtime/aws-runtime/src/lib.rs

```diff
@@ -38,6 +38,9 @@ pub mod invocation_id;
 /// Supporting code for request metadata headers in the AWS SDK.
 pub mod request_info;
 
+#[doc(hidden)]
+pub mod sdk_feature;
+
```

Comment:
Though hidden, I still think we should document this for ourselves.

Reply:
Makes sense. Will do the same for the `sdk_feature` module in the `aws-smithy-runtime` crate.